### PR TITLE
Add more docs to phx.gen.auth tokens

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

File: priv/templates/phx.gen.auth/schema_token.ex [hunk 1]
```diff
@@ -27,6 +27,20 @@ defmodule <%= inspect schema.module %>Token do
...
+  Phoenix' default session cookies are not persisted, they are
+  simply signed and potentially encrypted. This means they are
+  valid indefinitely, unless you change the signing/encryption
+  salt.
+
+  Therefore, storing them allows individual <%= schema.singular %>
+  sessions to be expired. The token system can also be extended
+  to store additional data, such as the device used for logging in.
+  You could then use this information to display all valid sessions
+  and devices in the UI and allow users explicitly expire any
```
Comment (nickjj) [inline priv/templates/phx.gen.auth/schema_token.ex:42]: > and allow users explicitly expire

This should likely be changed to: *and allow users to explicitly expire*

**Review by nickjj — CHANGES_REQUESTED** [review]
Looks good minus the 1 grammar issue.

File: priv/templates/phx.gen.auth/schema_token.ex (hunk 1 above)
Comment (josevalim) [inline priv/templates/phx.gen.auth/schema_token.ex:42]:
```suggestion
  and devices in the UI and allow users to explicitly expire any
```

**Review by nickjj — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**